lib/backend_common/backend_common/auth.py

```diff
@@ -150,7 +150,7 @@ def __init__(self, token, userinfo):
         self.token = token
         self.userinfo = userinfo
 
-        logger.info('Init user {}'.format(self.get_id()))
+        logger.info(f'Init user {self.get_id()}')
```

Comment:
I don't like seeing a function call in the f-string (from a style point of view), looks sketchy and less readable. What do you think @marco-c  ?

Reply:
Yeah, agreed.